packages/api/src/models/db.ts

```diff
@@ -81,9 +81,11 @@ async function initDB(): Promise<void> {
   const logDBOperations = Config.isCloudEnv() ? false : true;
   const dbPoolSettings = getDbPoolSettings();
 
-  docTableNames = {
-    token: tokenTableName,
-  };
+  if (tokenTableName) {
+    docTableNames = {
+      token: tokenTableName,
+    };
+  }
```

Comment:
We should update the type of `docTableNames` so its `DocTableNames | undefined` (and initialize it as `undefined` by default. This will prob mean changes downstream.

Reply:
that's a good way to incur the compiler checks! Thanks!